### PR TITLE
Add missing context validation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: program_budget/account_analytic_account.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def create(self, cr, uid, vals, context=None):
 
         Create the related result if this account is under
         the root program node"""
+        context = context or {}
```
Comment (pedrobaeza) [inline program_budget/account_analytic_account.py]: I prefer to not lose previous empty context: `context = {} if context is None else context`

**Comment by ghost** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

**Comment by joaoalf** [comment]
:+1:

**Pull request merged**